fe/fe-core/src/main/java/com/starrocks/external/hive/Utils.java

```diff
@@ -143,4 +143,11 @@ public static String getTypeKeyword(String type) {
         }
         return keyword;
     }
+
+    // Decimal string like "Decimal(3,2)"
+    public static int[] getPrecisionAndScale(String typeStr) {
+        String partWithComma = typeStr.substring(8, typeStr.length() - 1);
```

Comment:
using a regex match group is more rebust

"decimal\s*\((\d+)\s*,\s*(\d+)\)"
group(1)->precision
group(2)->scale

Add UT for this function